tasks.py

```diff
@@ -0,0 +1,83 @@
+from robocorp.tasks import task
+from robocorp import windows, log
+from robocorp.windows import WindowElement
+from RPA.Desktop import Desktop
+import os
+
+def open_calculator() -> WindowElement:
+    windows.desktop().windows_run("calc.exe")
+    window = windows.find_window('regex:.*Calculator')
+    return window
+
+def calculate_using_send_keys(window: WindowElement, first: str, second: str ):
+    """
+    Automation using send_keys() and hotkeys provided by the application is solid and by far the fastest option
+    ...if the target app provides keyboard shortcuts
+    """
+    window.send_keys('{BACK}{BACK}{BACK}{BACK}')
+    window.send_keys(f'{first}+{second}=')
+
+def calculate_using_ids(window: WindowElement, first: str, second: str ):
+    """
+    Automation using locators based on automation id -field values is a solid way
+    ...if the target app provides static automation ids
+    """
+    window.click('id: "clearEntryButton" or name: "Clear"')
+    window.click(f'id: "num{first}Button" or name: "{first}"')
+    window.click('id: "plusButton" or name: "Add"')
+    window.click(f'id: "num{second}Button" or name: "{second}"')
+    window.click('id: "equalButton" or name: "Equals"')
+
+def calculate_using_image_locators(window: WindowElement, first: str, second: str):
+    """
+    Automation using image-based locators is the slowest and most brittle form, but sometimes still needed.
+    Image recognition takes time and resources
+    Changes to resolution, themes and UI changes all can break the automation.
+
+    """
+    try:
+        window.click('id: "clearEntryButton" or name: "Clear"')
+        desktop = Desktop()
+        desktop.click(f'alias:{first}')
+        desktop.click('alias:plus')
+        desktop.click(f'alias:{second}')
+        desktop.click('alias:eq')
+    except Exception:
+        # Image-based solution fails on most machines, so we catch 
+        # the error and add it to the log, but do not fail the execution.
+        # Normal operation here should be to raise the exception here or not to catch it.
+        log.critical('Image-based locator not found')
```

Comment:
```suggestion
    except Exception as exc:
        # Image-based solution fails on most machines, so we catch 
        # the error and add it to the log, but do not fail the execution.
        # Normal operation here should be to raise the exception here or not to catch it.
        log.critical(f'Image-based locator not found due to {exc!r}')
```

In case you want to give more details about the error and in case that's not appearing in the logs by default during the catch.